app/main.py

```diff
@@ -2,0 +1,25 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    # Check each friend using try/except
+    for friend in friends:
+        try:
+            # Try to allow the friend to visit the cafe
+            cafe.visit_cafe(friend)
+        except VaccineError:  # Removed 'as ve'
+            # Catch both NotVaccinatedError and OutdatedVaccineError
+            return "All friends should be vaccinated"
```

Comment:
Code Style: It's recommended to use descriptive error messages. In this case, you are returning a string which is a good practice, but the message could be more descriptive about the issue. Maybe something like: 'One of your friends is not vaccinated. Everyone should be vaccinated to go to the cafe.'